worker/agentconfigupdater/manifold.go

```diff
@@ -73,6 +84,30 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, dependency.ErrUninstall
 			}
 
+			// Get controller config.
+			var dbGetter changestream.WatchableDBGetter
+			if err := context.Get(config.ChangeStreamName, &dbGetter); err != nil {
+				return nil, errors.Trace(err)
```

Comment:
Would it be handy to annotate the error with the ChangeStreamName? Or is that covered elsewhere?